src/tracelib/trace.ts

```diff
@@ -9,6 +9,8 @@
  */
 
 import {Dictionary, Producer, Predicate} from '../runtime/hot.js';
+import {now as nowMillis} from '../platform/date-web.js';
+import {pid} from '../platform/process-web.js';
```

Comment:
is `pid` still used anywhere?

Reply:
yes, it's used below. I believe it's required in the message format for the traces to be opened in the chrome tracing view.